src/Bridge/TracerBridge.php

```diff
@@ -12,6 +13,9 @@ class TracerBridge implements EventSubscriberInterface
 {
     private $tracer;
 
+    /**
+     * @param FlushableInterface & ResettableInterface $tracer
```

Comment:
```suggestion
```

The narrowing type here can be considered as a BC break.
Added code below ready to use `$tracer` that doesn't implement `ResettableInterface`, so let's don't change type expectations here.